src/rust/fr.lecercleduscan/src/parser.rs

```diff
@@ -0,0 +1,186 @@
+use aidoku::{
+	prelude::*,
+	error::Result,
+	std::{
+		html::Node,
+		String, StringRef, Vec, current_date, json
+	},
+	Manga, Page, MangaPageResult, MangaStatus, MangaContentRating, MangaViewer, Chapter
+};
+
+pub fn parse_manga_list(html: Node) -> Result<MangaPageResult>  {
+	let mut mangas: Vec<Manga> = Vec::new();
+
+	for page in html.select(".group").array() {
+		let page = page
+			.as_node()
+			.expect("Failed to get data as array of nodes");
+		
+		let title = page.select(">.title a").attr("title").read();
+		let url = page.select(">.title a").attr("href").read();
+		let id = String::from(url.split('/').enumerate().nth(4).expect("Failed to get id").1.trim());
+		let cover = String::from(page.select(".preview").attr("src").read().trim());
+
+		mangas.push(Manga {
+			id,
+			cover,
+			title,
+			author: String::new(),
+			artist: String::new(),
+			description: String::new(),
+			url,
+			categories: Vec::new(),
+			status: MangaStatus::Unknown,
+			nsfw: MangaContentRating::Safe,
+			viewer: MangaViewer::Rtl
+		});
```

Comment:
use `Default::default()` instead of adding in empty strings https://github.com/Skittyblock/aidoku-community-sources/blob/c66604ab3c956daabd038a64a9c06f58879711a2/src/rust/en.tsumino/src/parser.rs#L19-L27